PotreeConverterPartitioned/scheduler/localScheduler.py

```diff
@@ -0,0 +1,79 @@
+import subprocess
+from multiprocessing import Process
+import psutil
+from scheduler import Scheduler
+
+from ..loggingwrapper import LoggingWrapper
+
+
+
+
+class LocalScheduler(Scheduler):
+    """Class for local job scheduler"""
+    def runProgram(self):
+        programCommandStatus = subprocess.run(self.programCommand, shell=True,capture_output=True, encoding="utf-8")
```

Comment:
is the shell=True option necessary at this point. Maybe largely moot, but as the programCommand is fully user configurable this would be classed as a shell injection vunerability (although if the user is running the code on a system they hhave access to anyway it doesn't matter. This might be an issue if the converter were exposed via an online interface.

Reply:
You are right. "Args" in subprocess.run should be a list but I am passing it a string which forcede me to use shell=True. It requires a bit of code refactoring to replace all strings args to list.